lib/kitchen/directions/bake_notes/bake_autotitled_note.rb

```diff
@@ -2,8 +2,16 @@
 
 module Kitchen
   module Directions
-    module BakeTitledNote
-      def self.v1(note:)
+    module BakeAutotitledNote
+      def self.v1(book:, selectors:)
```

Comment:
the second argument can't be any arbitrary selectors because you are comparing against classes.  So I'd call it `classes`